TravelAgencyService/Services/AgencyOperationsService.cs

```diff
@@ -0,0 +1,52 @@
+using TravelAgencyService.Models;
+
+namespace TravelAgencyService.Services;
+
+public class AgencyOperationsService
+{
+       
+    public void BookTicket(AgencyModel travelAgency)
+    {
+        Console.Write("Enter the name of the destination: ");
```

Comment:
I would move all the console operations to separate class and leave only business logic here